app/src/main/res/raw/plantdata.csv

```diff
@@ -1,4 +1,4 @@
-;Kategori;Tidligste såning/plantning;Seneste såning/plantning;Plantes eller sås;Sædskifteinterval;Antal frø/planter pr. meter række;Sådybde;Afstand mellem rækkerne i cm;Gødningsniveau;Forventet høst per række
+;Kategori;Tidligste såning/plantning;Seneste såning/plantning;Plantes eller sås;Sædskifteinterval;Antal frø/planter pr. meter række;Sådybde i cm;Afstand mellem rækkerne i cm;Gødningsniveau;Forventet høst per række
```

Comment:
Why did you add this but no remove "cm" from all the data? "Sådybde i cm: 4 cm" doesn't add any value